src/psygnal/_group_descriptor.py

```diff
@@ -426,13 +536,15 @@ def __get__(
         if instance is None:
             return self
 
+        signal_group = self._get_signal_group(owner)
```

Comment:
I had to do this to make it work, but I think it makes sense in general so `signal_group` is not recomputed for each new instance. This should not affect much performances as the call to `_build_dataclass_signal_group` was cached anyway.